### PR TITLE
Gerty extension tweaks

### PR DESCRIPTION
**Pull request by blackcoffeexbt**
General improvements to the Gerty extension

### CODE REVIEW

File: lnbits/extensions/gerty/helpers.py [hunk 1]
```diff
@@ -41,17 +41,17 @@ def get_text_item_dict(
...
-            font_size = 1
-        if font_size <= 15:
-            font_size = 1
-        elif font_size <= 20:
-            font_size = 2
-        elif font_size <= 40:
-            font_size = 2
-        else:
-            font_size = 5
+    # if gerty_type.lower() == "mini gerty":
```
Comment (motorina0) [inline lnbits/extensions/gerty/helpers.py:44]: - dead code should be removed (if not planned to be re-enabled in the near future)

File: lnbits/extensions/gerty/helpers.py [hunk 2]
```diff
@@ -459,7 +459,9 @@ async def get_screen_data(screen_num: int, screens_list: list, gerty):
                     response = await client.get(url)
                     text.append(
                         get_text_item_dict(
-                            text=url,
+                            text=url.replace("https://", "")
```
Comment (motorina0) [inline lnbits/extensions/gerty/helpers.py:462]: nitpick: extract function for (it is used in two places):
```py
url.replace("https://", "")
            .replace("http://", "")
            .strip("/"),
```

File: lnbits/extensions/gerty/helpers.py [hunk 3]
```diff
@@ -765,17 +776,31 @@ async def get_onchain_dashboard(gerty):
     areas = []
     if isinstance(gerty.mempool_endpoint, str):
         async with httpx.AsyncClient() as client:
+            text = []
```
Comment (motorina0) [inline lnbits/extensions/gerty/helpers.py:779]: - can this new logic be one level up? Outside of:
```py
async with httpx.AsyncClient() as client:
```

File: lnbits/extensions/gerty/templates/gerty/index.html [hunk 4]
```diff
@@ -130,6 +129,35 @@ <h6 class="text-subtitle1 q-my-none">
...
+      <center>
+        <q-btn
+          dense
+          outline
+          unelevated
+          color="primary"
+          size="md"
+          @click="copyText(settingsDialog.apiUrl, 'Link copied to clipboard!')"
+          >{% raw %}{{settingsDialog.apiUrl}}{% endraw %}<q-tooltip>
+            Click to copy URL
```
Comment (motorina0) [inline lnbits/extensions/gerty/templates/gerty/index.html:151]: nitpick: `Copy URL`

File: lnbits/extensions/gerty/templates/gerty/index.html [hunk 5]
```diff
@@ -300,7 +335,12 @@ <h6 class="text-subtitle1 q-my-none">
     )
     obj.fsat = new Intl.NumberFormat(LOCALE).format(obj.amount)
     obj.gerty = ['/gerty/', obj.id].join('')
-    obj.gertyJson = ['/gerty/api/v1/gerty/pages/', obj.id, '/0'].join('')
+    obj.gertyJson = [
+      window.location.origin,
+      '/gerty/api/v1/gerty/pages/',
+      obj.id
+    ].join('')
+    //obj.gertyJson = new URL(['/gerty/api/v1/gerty/pages/', obj.id].join(''))
```
Comment (motorina0) [inline lnbits/extensions/gerty/templates/gerty/index.html:343]: - remove dead code

File: lnbits/extensions/gerty/helpers.py (hunk 3 above)
Comment (blackcoffeexbt) [reply lnbits/extensions/gerty/helpers.py:779]: Thanks for the review Vlad. I'm not sure what you mean by this comment? Could you elaborate please and I'll address?

File: lnbits/extensions/gerty/views_api.py [hunk 6]
```diff
@@ -93,6 +93,13 @@ async def api_gerty_satoshi():
     return await get_satoshi()
 
 
+@gerty_ext.get("/api/v1/gerty/pages/{gerty_id}")
```
Comment (motorina0) [inline lnbits/extensions/gerty/views_api.py:96]: - is this just a place-holder for future logic to be added?

File: lnbits/extensions/gerty/helpers.py (hunk 3 above)
Comment (motorina0) [reply lnbits/extensions/gerty/helpers.py:779]: hmm, I see now that `client` variable is not used.
So the bleow line can be removed entirely (and also reduce one level of nesting):
```py
async with httpx.AsyncClient() as client:
```

- same applies for the `client` variable in `get_onchain_stat`

File: lnbits/extensions/gerty/views_api.py (hunk 6 above)
Comment (blackcoffeexbt) [reply lnbits/extensions/gerty/views_api.py:96]: Looks like it. Removed for now.

File: lnbits/extensions/gerty/templates/gerty/index.html (hunk 5 above)
Comment (blackcoffeexbt) [reply lnbits/extensions/gerty/templates/gerty/index.html]: Done

File: lnbits/extensions/gerty/helpers.py (hunk 3 above)
Comment (blackcoffeexbt) [reply lnbits/extensions/gerty/helpers.py]: Done

**Comment by blackcoffeexbt** [comment]
Feedback from Vlad's review implemented and pushed.

**Pull request merged**